docs/package.json

```diff
@@ -17,15 +17,18 @@
     "test": "jest",
     "schema-sync": "node --async-stack-traces --unhandled-rejections=strict ./scripts/schema-sync.js"
   },
+  "resolutions": {
+    "webpack": "^5.28.0"
```

Comment:
re: "I have also reverted the removal of webpack resolutions in package.json (#12794). Without this entry webpack have been removed from yarn.lock after first dependency change".

Thank you for noticing this! That's good to know.

If we keep this I think we should change it to `"webpack": "^5"`

Reply:
LGTM! Thanks!